kufar/books/main.go

```diff
@@ -342,6 +323,36 @@ func ReadBooks(d Database) func(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (c Cache) get(id int) (Book, bool) {
+	var bk Book
+
+	c.mu.RLock()
```

Comment:
Please use `defer c.mu.RUnlock()` in functions with several possible returns. It will help you not to forget to unlock mutex if you will add some new return path :)